draft-dekater-scion-pki.md

```diff
@@ -1552,3 +1555,34 @@ Phase 3 concludes after all voting representatives have cast their votes.
 In Phase 4, all voting representatives share the signed TRC with the ceremony administrator. This happens again over the data exchange device, which goes from one voter to the next. Each voting representative copies the TRC payload signed with the voter's private keys from their own machine onto the data exchange device. The last voter returns the device to the ceremony administrator, who assembles the final TRC by aggregating the payload data with the votes (signatures) cast by the voting representatives.
 
 The signed TRC is validated by inspecting its contents on the monitor and verifying the signatures based on the exchanged certificates in Phase 1. The ceremony administrator then shares the signed TRC with all participants. Each of them MUST then inspect it once more, and verify it based on the certificates exchanged in Phase 1. At this point, the ceremony is completed. All participants have the signed TRC, and can use it to distribute the trust anchors for their ISD.
+
+
+# Change Log
+{:numbered="false"}
+
+Changes made to drafts since ISE submission. This section is to be removed before publication.
+
+## draft-dekater-scion-pki-07
+{:numbered="false"}
+
+Minor changes:
+
+- Added Ceremony & Deployment process.
+- Added Trusted Root Configuration (TRC) Specification.
+- Added Certificate Specification.
```

Comment:
Where did we add these?  Wasn't this a while ago (before the ISE submission)?
```suggestion
```

Reply:
The Ceremony & Deployment process was updated since the ISE submission. I don't remember when the other stuff was added.